src/cmd/manage_nsfs.js

```diff
@@ -572,7 +571,8 @@ async function manage_account_operations(action, data, show_secrets, user_input)
         await delete_account(data);
     } else if (action === ACTIONS.LIST) {
         const account_filters = _.pick(user_input, LIST_ACCOUNT_FILTERS);
-        const accounts = await list_config_files(TYPES.ACCOUNT, accounts_dir_path, data.wide, show_secrets, account_filters);
+        const wide = get_boolean_or_string_value(user_input.wide);
```

Comment:
@shirady do we need this check here? `_.isUndefined(user_input.wide)`

Reply:
@naveenpaul1 I don't need this check because when the value is `undefined` then the function `get_boolean_or_string_value` will return `false`.
https://github.com/noobaa/noobaa-core/blob/83166c69ab308d7b37b2656a5561c972ca720889/src/cmd/manage_nsfs.js#L1042-L1044
Before I had to keep it as `undefined` because it was part of the data and it would add `wide: false` to the bucket and the account and would fail the schema validation.
It was part of the motivation for opening the issue.